Library/include/CSP/Systems/ECommerce/ECommerceSystem.h

```diff
@@ -106,6 +106,16 @@ class CSP_API ECommerceSystem : public SystemBase
 										  const common::String& PrivateAccessToken,
 										  AddShopifyStoreResultCallback Callback);
 
+	/// @brief Sets a store to be enabled or disabled in a space.
+	/// @param StoreName csp::common::String : The store name (URL) to the Shopify store. Do not include the '.shopify.com' part of the url.
+	/// @param SpaceId csp::common::String : ID of the space the store links to.
+	/// @param IsEcommerceActive bool : Bool to set the ecommerce system status to.
+	/// @param Callback SetECommerceEnabledResultCallback : Callback when asynchronous task finishes
+	CSP_ASYNC_RESULT void SetECommerceEnabledInSpace(const common::String& StoreName,
+													 const common::String& SpaceId,
+													 const bool IsEcommerceActive,
```

Comment:
Nitpick; could we use the same language as the function name (ie adopt either 'Active' or 'Enabled' for both)?

Reply:
Yeah that's fine